34.md

```diff
@@ -87,8 +87,9 @@ Replies are also Markdown text. The difference is that they MUST refer to one ro
 }
 ```
 
-## Things to be added later
+## Possible things to be added later
 
-- branch merge kind (specifying a URL from where to fetch the branch to be merged)
-- inline file comments kind
-- review type with multiple inline comments? -- or maybe this could just be an extension of the "reply" kind above
+- "status" kind (for letting people know a patch was merged or an issue was fixed or won't be fixed)
```

Comment:
Status is quite important. Shouldn't we add this now? Its impossible to identify open proposals or issues unless you are a maintainer with a client that stores the status internally.